packages/utils/src/validators.ts

```diff
@@ -50,16 +49,16 @@ function findSeatPriceForProtocolBefore49(validators: (CurrentEpochValidatorInfo
 }
 
 // nearcore reference: https://github.com/near/nearcore/blob/5a8ae263ec07930cd34d0dcf5bcee250c67c02aa/chain/epoch_manager/src/validator_selection.rs#L308;L315
-function findSeatPriceForProtocolAfter49(validators: (CurrentEpochValidatorInfo | NextEpochValidatorInfo)[], maxNumberOfSeats: number, minimumStakeRatio: number[]): BN {
+function findSeatPriceForProtocolAfter49(validators: (CurrentEpochValidatorInfo | NextEpochValidatorInfo)[], maxNumberOfSeats: number, minimumStakeRatio: number[]): bigint {
     if (minimumStakeRatio.length != 2) {
         throw Error('minimumStakeRatio should have 2 elements');
     }
-    const stakes = validators.map(v => new BN(v.stake, 10)).sort((a, b) => a.cmp(b));
-    const stakesSum = stakes.reduce((a, b) => a.add(b));
+    const stakes = validators.map(v => BigInt(v.stake)).sort((a, b) => (a < b ? -1 : a > b ? 1 : 0));
```

Comment:
Since ordering of bigints is used in two places already I think it's better to create a util file with a function that sorts it. It's will be much more understandable